tests/test_glacier.py

```diff
@@ -257,3 +257,23 @@ def test_glacier_remove_statement(self):
             ]
         )
         self.assertTrue("RemoveMe" not in [s["Sid"] for s in data.get("Statement", ())])
+
+
+class GlacierVaultTest(BaseTest):
+
+    def test_glacier_vault_delete(self):
+        session_factory = self.replay_flight_data("test_glacier_vault_delete")
+        p = self.load_policy(
+            {
+                "name": "glacier-vault-delete",
+                "resource": "aws.glacier",
+                "filters": [{"type": "value", "key": "VaultName", "value": "c7n-test-delete"}],
+                "actions": [{"type": "delete"}],
+            },
+            session_factory=session_factory,)
+        resources = p.run()
+        self.assertEqual(len(resources), 1)
+        client = session_factory().client("glacier")
+        response = client.list_vaults()
+        self.assertEqual(response.get(
```

Comment:
Does it make sense to assert that the deleted vault name is not in the list here instead? The goal seems to be to demonstrate that it no longer exists, correct?

Reply:
yes it does and addressed! Thanks!